### PR TITLE
handle space-separated domain

### PR DESCRIPTION
**Pull request by leftstick**
The space-separated domain cannot be handled correctly in previous version, for example:

If i have an host rule like: 

```
127.0.0.5 www.peercdn.com  m.peercdn.com
```

The host rule gets matched looks like:

```javascript
['127.0.0.5 www.peercdn.com', 'm.peercdn.com']
```

Which is incorrect obviously.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
   "name": "hostile",
   "description": "Simple /etc/hosts manipulation",
-  "version": "1.1.1",
+  "version": "1.1.2",
```
Comment (feross) [inline package.json]: Don't increment the version in PRs. That's done at release time

File: .gitignore [hunk 2]
```diff
@@ -0,0 +1,2 @@
+#npm
+node_modules/
```
Comment (feross) [inline .gitignore]: Please don't commit files that are unrelated to the PR

**Comment by feross** [comment]
Thanks for the PR, I made the necessary fixes. Released as 1.2.0.

**Comment by leftstick** [comment]
Good to know, thanks for the review
